test/stub_composition_test.py

```diff
@@ -0,0 +1,10 @@
+# Copyright Modal Labs 2024
+from test.helpers import deploy_stub_externally
+
+
+def test_stub_composition_includes_all_functions(servicer, supports_dir, monkeypatch, client):
+    print(deploy_stub_externally(servicer, "main.py", cwd=supports_dir / "multifile_project"))
```

Comment:
Intentional?

Reply:
Not really, I think it was mostly for debugging